CITATION.cff

```diff
@@ -0,0 +1,10 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+  - name: "Nebari Development Team"
```

Comment:
@pavithraes I'm not familiar with the `.cff` syntax but looking at the [docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files#citation-formats), it seems one should pass `family-names` and/or `given-names`. Looking at the [pandas example](https://github.com/pandas-dev/pandas/blob/d62d77bb424a466619c2f7ace79e1a9c7732c562/CITATION.cff#L21), it's also using the latter syntax. If passing `name` works, I don't mind leaving it like it is now.

Otherwise, it looks good to me. Thank you!

Reply:
Thanks for taking a look. IIUC, the `family-names` and `given-names` syntax is for specific individuals, and we can use `name` to refer to a group. Ref: [Line #9 in this examples](https://github.com/citation-file-format/citation-file-format/blob/main/README.md#structure)